src/scripts/main.js

```diff
@@ -1 +1,45 @@
 'use strict';
+
+function toggleScroll() {
+  const body = document.body;
+  const menuComputedStyle = window.getComputedStyle(menu);
+  const productComputedStyle = window.getComputedStyle(product);
+
+  if (menuComputedStyle.opacity === '1'
+  || productComputedStyle.opacity === '1') {
+    body.classList.add('no-scroll');
+  } else {
+    body.classList.remove('no-scroll');
+  }
+}
+
+const menu = document.getElementById('menu');
+const product = document.getElementById('product');
+
+function checkScreenWidth() {
+  const screenWidth = window.innerWidth
```

Comment:
Both window.innerWidth and document.documentElement.clientWidth are used to get the screen width. It's recommended to stick with one to ensure consistent results.